.github/workflows/railway-deploy.yml

```
@@ -0,0 +1,188 @@
+        run: npm install -g @railway/cli
+
+      - name: 🚀 Deploy to Railway (Staging)
+        env:
+          RAILWAY_TOKEN: ${{ secrets.RAILWAY_TOKEN_STAGING }}
+        run: railway up --service staging
+
+      - name: ⏳ Wait for deployment
+        run: sleep 30
+
+      - name: 🏥 Health check
+        run: |
```

actionlint (1.7.10)

[error] shellcheck reported issue in this script: SC2086:info:2:6: Double quote to prevent globbing and word splitting

(shellcheck)